README.md

```diff
@@ -45,9 +48,19 @@ BioSignal Web Recorder is an application based on Web Serial connection, you can
 - [Tailwind CSS](https://tailwindcss.com/): Utility-first CSS framework.
 - [Shadcn UI](https://tailwindcss.com/): Provides Reusable and customizable components, built with radix UI.
 - [Lodash](https://lodash.com/): Utility library for data manipulation, used for data throttling.
-- [JSZip](https://stuk.github.io/jszip/): Library for creating and manipulating ZIP files, used for save recordings as ZIP.
 - [Smoothie Js](http://smoothiecharts.org/): Real time charting library.
+- [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API) Browser-based database for storing recorded data.
 
 ## Resources
 
 - [Vendors](https://github.com/akadeepesh/biosignal-recorder-web-private/blob/master/src/components/vendors.ts) list for board name and there vendor id are taken from [Stackblitz](https://stackblitz.com/edit/typescript-web-serial?file=vendors.ts) created by [William Grasel](https://github.com/willgm)
+
+## Contributors
+
+We are thankful to our awesome contributors, the list below is alphabetically sorted.
+
+- [Aman Maheswari](https://github.com/Amanmahe)
+- [Deepak Khatri](https://github.com/lorforlinux)
+- [Deepesh Kumar](https://github.com/akadeepesh)
+- [Mahesh Tupe](https://github.com/Asc91)
+- [Ritika Mishra](https://github.com/ritika8081)
```

Comment:
Will change this to github auto contributors fetch link

Reply:
Thank you for contributing to our project! Your support is invaluable in creating & enhancing Chords-Web and making it even better. 😊

<center>
<a href="https://github.com/upsidedownlabs/Chords-Web/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=upsidedownlabs/Chords-Web" />
</a>
</center>

Reply:
okay